deposit-app/src/main/java/edu/unc/lib/boxc/deposit/validate/VirusScanJob.java

```diff
@@ -42,6 +42,7 @@ public class VirusScanJob extends AbstractConcurrentDepositJob {
             .getLogger(VirusScanJob.class);
 
     private static final int MAX_RETRIES = 5;
+    private long maxStreamSize = 0;
```

Comment:
is 0 the default?

Reply:
Yes, which would mean it'd send everything to streaming